Reflect release 5.7.2

matriv opened the pull request.

BaurzhanSakhariev commented:
I think commit message should be reflect relase 5.7.2?

It also means that https://github.com/crate/crate/pull/15957 got a wrong commit message 😢

matriv commented:
> I think commit message should be reflect relase 5.7.2?
> 
> It also means that #15957 got a wrong commit message 😢

Thx, so much, fixed, for #15957 at least I fixed the PR title.
